contributor_docs/documentation_style_guide.md

```diff
@@ -0,0 +1,1184 @@
+}
+```
+
+Communicate the current way of doing things, both explicitly and implicitly. Use the idioms recommended in this guide. Reorder sections to emphasize favored approaches if needed. The documentation should be a model for best practices and approachable for beginners.
+
+Documentation has to be brief but comprehensive. Explore and document edge cases. What happens for each combination of arguments? What bugs are most likely to appear in a beginner's code?
+
+Spell names correctly: p5.js, CSS, HTML, JavaScript, WebGL. When in doubt, refer to an authoritative source like their official documentation.
+
+**[⬆ back to top](#table-of-contents)**
+
+## Unbiased Documentation
+
+Write documentation without bias towards any kind of person. While documenting particularly demanding/sensitive topics, take the time to educate yourself. Ensure that your writing doesn’t hurt or offend someone unintentionally.
+
+While writing unbiased documentation:
+
+- Be inclusive of every gender identity and expression, sexual orientation, race, ethnicity, language, neuro-type, size, disability, class, religion, culture, subculture, political opinion, age, skill level, occupation, and background. Make examples as diverse as our community.
+- Avoid politicized content. If political content is necessary, remain neutral.
+- Follow accessibility guidelines.
+- Avoid content that would insult or cause harm to people.
+- Don’t make any generalizations about people, countries, and cultures. That includes positive or neutral generalizations.
+- Don’t write prejudiced and discriminatory content against minority communities.
+- Avoid terms related to historical events.
+
+Prefer wordings that avoid "you"s and "your"s. For example, instead of:
+
+```
+If you need to declare a variable, it is recommended that you use `const` or `let`.
+```
+
+use this style:
+
+```
+Use `const` or `let` to declare variables.
+```
+
+**Pronouns**
+
+| Recommended |	Not Recommended |
+| -- | -- |
+| they | he or she |
+| them | him or her |
+| their | his or her |
+| theirs | his or hers |
+| themselves | himself or herself |
+
+**[⬆ back to top](#table-of-contents)**
+
+## Accessibility and Disability
+
+- Emphasize the reader rather than underlining their inconveniences.
+- Don’t refer to a person with a disability as a disabled person. Use approved terminology for people with specific disabilities.
+- Maintain a uniform structure throughout the p5.js documentation. Emphasize important points both stylistically and visually.
+- Use a screen reader to test documentation. To test a screen reader, see [List of screen readers](https://en.wikipedia.org/wiki/List_of_screen_readers).
+- Consider multi-platform accessibility for all types of devices and operating systems.
+- Create examples with all types of input devices such as voice and gesture based devices, controllers, mice, and keyboards.
+- Don’t use ableist language. Be inclusive and unbiased while writing about accessibility and disability.
```

Comment:
Maybe we can include a link on guidelines for avoiding the use of ableist language?

Reply:
Great suggestion. I'll adapt this [guide from the National Library of Medicine](https://news.nnlm.gov/region_7/2022/08/01/what-is-ableism-and-how-is-it-a-part-of-the-everyday-language-we-use-at-the-library/) and include a link.